spec/models/download_spec.rb

```diff
@@ -99,35 +99,53 @@
   describe "authors" do
     let(:work) { Work.new }
     let(:subject) { Download.new(work) }
+    let(:pseud1) { create(:pseud, name: "First", user: create(:user)) }
+    let(:pseud2) { create(:pseud, name: "Second", user: create(:user)) }
 
     it "joins the author names separated by a comma and a space" do
```

Comment:
You can remove this test, since it's checking the same thing as "for a work with multiple authors" below.